opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -370,12 +370,14 @@ mod tests {
     #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
     async fn counter_aggregation_overflow_delta() {
         counter_aggregation_overflow_helper(Temporality::Delta);
+        counter_aggregation_overflow_helper_custom_limit(Temporality::Delta);
     }
 
     #[ignore = "https://github.com/open-telemetry/opentelemetry-rust/issues/1065"]
```

Comment:
Remove the ignore attribute from `counter_aggregation_overflow_cumulative` and `counter_aggregation_overflow_delta` tests now that you're adding cardinality limit.

Reply:
yes good catch!